src/main/frontend/app/components/NavBar.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react'
+import './NavBar.scss'
+
+export default class NavBar extends Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            btn1: 'Sign-Up',
+            ref1: '/registration',
+            btn2: 'Log-In',
+            ref2: '/login'
+        }
+    }
+
+    componentDidMount(){
+        
+        fetch("http://localhost:8080/api/v1/currentLoginUser")
+        .then(response => {
+            if (response.ok) {
+              return response.json();
+            } else {
+              throw new Error('Something went wrong ...');
+            }
+          })
+          .then(data => this.setState({  
+              btn1: 'Profile',
+          ref1: '/welcome',
+          btn2: 'Log-out',
+          ref2: '/login' }))
+          .catch(error => this.setState({ 
+              btn1: 'Sign-Up',
+          ref1: '/registration',
+          btn2: 'Log-In',
+          ref2: '/login'}));  
+    }
+
+    render() {
+        
```

Comment:
when there is nothing in the render, the return should be on the very next line.